python/README.md

```diff
@@ -0,0 +1,67 @@
+# OpenTracing support for gRPC in Python
+
+The `grpcio-opentracing` package makes it easy to add OpenTracing support to 
+gRPC-based systems in Python.
+
+## Installation
+
+```
+pip install grpcio-opentracing
+```
+
+## Getting started
+
+See the below code for basic usage or [examples/trivial](examples/trivial) for a
+complete example.
+
+### Client-side usage example
+
+```python
+tracer = # some OpenTracing Tracer instance
+interceptor = open_tracing_client_interceptor(tracer)
```

Comment:
why is there this helper function instead of just initializing an OpenTracingClientInterceptor directly?